locale/tr/LC_MESSAGES/More/EAV/README.po

```diff
@@ -17,7 +17,7 @@ msgstr "Entity-Attribute-Value (Varlık-Özellik-Değer)"
 
 #: ../../More/EAV/README.rst:4
 msgid "The Entity–attribute–value (EAV) pattern in order to implement EAV model with PHP."
-msgstr ""
+msgstr "-"
```

Comment:
Why "-"?

Reply:
'Cos it does not add anything to translation. It sounds EAV is EAV.

Reply:
@k-gun first of all. If you put "-" in msgstr then this whole sentence "The Entity–attribute–value (EAV) pattern in order to implement EAV model with PHP." will be translated into "-" - this is not cool. Other thing is that this msgid is not a shortcut only but full sentence that should be translated as well.